modules/tinymce/src/core/test/ts/webdriver/PageUpDownKeysTest.ts

```diff
@@ -0,0 +1,98 @@
+      const offset = platform.browser.isIE() ? 5 : 4;
+      TinySelections.setCursor(editor, [ 0, 0, 0 ], offset);
+      await RealKeys.pSendKeysOn('iframe => body', [ RealKeys.combo({}, 'PageUp') ]);
+
+      TinyAssertions.assertCursor(editor, [ 0 ], 0);
+    });
+
+    it('TINY-4612: caret wont move if it is not at the beginning/end of the inline element', async () => {
+      const editor = hook.editor();
+      editor.setContent('<p>text<a href="google.com">link</a>link</p>');
+      TinySelections.setCursor(editor, [ 0, 1, 0 ], 2);
+      await RealKeys.pSendKeysOn('iframe => body', [ RealKeys.combo({}, 'PageUp') ]);
+
+      TinyAssertions.assertCursor(editor, [ 0, 1, 0 ], 2);
+    });
+
+    it('TINY-4612: caret wont move inline_boundaries: false', async () => {
+      const editor = hook.editor();
+      editor.settings.inline_boundaries = false;
+      editor.setContent('<p>text<a href="google.com">link</a>link</p>');
+      TinySelections.setCursor(editor, [ 0, 1, 0 ], 0);
+      await RealKeys.pSendKeysOn('iframe => body', [ RealKeys.combo({}, 'PageUp') ]);
+
+      TinySelections.setCursor(editor, [ 0, 1, 0 ], 0);
+      delete editor.settings.inline_boundaries;
+    });
+  });
+
+  context('Page Down', () => {
+    it('TINY-4612: caret should be placed after the inline element if it is at the start of it', async () => {
+      const editor = hook.editor();
+      editor.setContent('<p><a href="google.com">link</a>text</p>');
+      TinySelections.setCursor(editor, [ 0, 0, 0 ], 0);
+      await RealKeys.pSendKeysOn('iframe => body', [ RealKeys.combo({}, 'PageDown') ]);
+
+      TinyAssertions.assertCursor(editor, [ 0 ], 1);
+    });
+
+    it('TINY-4612: caret should be placed after the inline element if it is at the end of it', async () => {
+      const editor = hook.editor();
+      editor.setContent('<p>text<a href="google.com">link</a></p>');
+      TinySelections.setCursor(editor, [ 0, 1, 0 ], 4);
+      await RealKeys.pSendKeysOn('iframe => body', [ RealKeys.combo({}, 'PageDown') ]);
+
+      TinyAssertions.assertCursor(editor, [ 0 ], 2);
+    });
+
+    it('TINY-4612: caret wont move if it is not at the beginning/end of the inline element', async () => {
+      const editor = hook.editor();
+      editor.setContent('<p><a href="google.com">link</a>text</p>');
+      TinySelections.setCursor(editor, [ 0, 0, 0 ], 2);
+      await RealKeys.pSendKeysOn('iframe => body', [ RealKeys.combo({}, 'PageDown') ]);
+
+      TinySelections.setCursor(editor, [ 0, 0, 0 ], 2);
+    });
+
+    it('TINY-4612: caret wont move inline_boundaries: false', async () => {
+      const editor = hook.editor();
+      editor.settings.inline_boundaries = false;
+      editor.setContent('<p><a href="google.com">link</a>text</p>');
+      TinySelections.setCursor(editor, [ 0, 0, 0 ], 0);
+      await RealKeys.pSendKeysOn('iframe => body', [ RealKeys.combo({}, 'PageDown') ]);
+
+      TinySelections.setCursor(editor, [ 0, 0, 0 ], 0);
```

Comment:
This should also be an assertCursor.

- [ ] Change to assertCursor

Reply:
Un-resolving at this hasn't been changed

Reply:
done